src/components/project-pages/ConsciousShoppingContent.tsx

```diff
@@ -46,10 +51,29 @@ const SmallImage = styled.img<LayoutProps & FlexboxProps & SpaceProps>`
   ${space};
 `;
 
+const Arrow = styled.img<LayoutProps & SpaceProps & PositionProps>`
+  ${layout};
+  ${space};
+  ${position};
+  z-index: ${zIndexes.inFront};
+  cursor: pointer;
+  transition: transform 0.2s;
+  transform-origin: left;
+  &:hover {
+    transform: scale(1.05);
+  }
+`;
+
+const TextContainer = styled.div`
+  overflow-y: scroll;
+  ::-webkit-scrollbar {
```

Comment:
why did you need to do this?

"add scroll", "hide scroll bar"
seems like bad UI

Reply:
Goood point

Reply:
Basically what I was trying to do was hide the scrollbar while maintaining the scroll functionality. This should be fixed now.